### PR TITLE
fixes test_positive_sync_plan_disable_enable

### PR DESCRIPTION
**Pull request by jameerpathan111**
Fixes issue #73

### CODE REVIEW

**Comment by ntkathole** [comment]
[test]

File: tests/conftest.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import datetime
-from fauxfactory import gen_string
 import pytest
+import yaml
```
Comment (ntkathole) [inline tests/conftest.py]: This will need pyyaml module. Do you want to add pyyaml dependency ?

**Comment by jameerpathan111** [comment]
Test Results:
using upstream foreman-maintain v0.3.x
```
$ pytest --ansible-host-pattern satellite --ansible-user=root  --ansible-inventory testfm/inventory tests/test_advanced.py -k test_positive_sync_plan_disable_enable -svvv
=================================================================================== test session starts ===================================================================================
collected 15 items / 14 deselected                                                                                                                                                        

tests/test_advanced.py::test_positive_sync_plan_disable_enable 2019-06-13 12:25:09,753 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
disable active sync plans: 
\ Total 3 sync plans are now disabled.                                [OK]      
--------------------------------------------------------------------------------
2019-06-13 12:25:50,601 - testfm.log - INFO - Running ForemanMaintain::Scenario
================================================================================
re-enable sync plans: 
- Total 3 sync plans are now enabled.                                 [OK]      
--------------------------------------------------------------------------------
PASSED

======================================================================== 1 passed, 14 deselected in 205.25 seconds ========================================================================
```

**Comment by ntkathole** [comment]
ok to test : tests/test_advanced.py -k test_positive_sync_plan_disable_enable

File: tests/conftest.py (hunk 1 above)
Comment (ntkathole) [reply tests/conftest.py]: Ignore this, it comes from ansible 
`Collecting PyYAML (from ansible==2.6.17->-r requirements.txt (line 1))`

**Pull request merged**